javascripts/main.js

```diff
@@ -0,0 +1,90 @@
+$(document).ready(function(){
+
+    var categories = [];
+    var types = [];
+    var products = [];
+
+    function makeDropdownLinks () {
+        var dropdownString = "";
+
+        for (var i=0; i < categories.length; i++) {
+            dropdownString += `<li><a href="#" id="category-${categories[i].id}" class="category">${categories[i].name}</a></li>`;
+        }
+        $("#choose-category").append(dropdownString);
+    }
+
+
+    $("#choose-category").on("click", ".category", function() {
+        var categoryId = $(this)[0].id;
+        var categoryIdNumber = categoryId.split("category-")[1];
```

Comment:
Try using this in place of what you have. 
 var categoryIdNumber = parseInt(categoryId.split("category-")[1]);

What is happening is in your if statement you are comparing an int and a string and these will never be equal so your if statement will never run. Let me know if you need anymore help.